### DISCUSSION TITLE
Yamaha AG06 audion interface

### DISCUSSION THREAD

**Discussion by Stewbow** [jamulussoftware/jamulus#3224] [General]
Hi Guys
I've played around unsuccessfully with my Yamaha AG06, tying to set it up so I have a Stereo Input and wanting to abide by The Golden Rule!!! But I can't find a way of Not Hearing Myself when I am am Not connected to Jamulus.
any ideas
thanks

**Comment by Jazzforever** [comment]
Hello,
having had a short look at the device, I propose: choose "Loopback" as "streaming out"-option and then mute both input channels (or one of them, if you are only using one). Now, direct monitoring should be disabled and, with the correct output-settings in Jamulus, you should only hear the signal from the computer.

**Comment by Stewbow** [comment]
Hi

I cant see a mute buttons on the input channels!!

But if I turn down the dials how is my music keyboard going to be heard by myself and anybody else?

 

Stuart

**Comment by henkdegroot** [comment]
> But I can't find a way of Not Hearing Myself when I am am Not connected to Jamulus.

When you are not connected to Jamulus, it is correct that you can't hear yourself! As you would listen to the audio from the server (which isn't there). So you seem to follow the rule correct.

**Reply by henkdegroot** [reply]
Obvious I should have read you post better... sorry

**Comment by Jazzforever** [comment]
Oh, you are using the keyboard input channel 3/4? You may connect your keyboard to the channels 1 and 2 (they have combined inputs for xlr- and mono jacks). Those channels have a red mute button. It is necessary to disable direct monitoring to avoid to hear yourself twice: through direct monitoring (to be avoided) and - with some delay - as the jamulus return via your computer, this is my understanding of the golden rule. Seems like with the Yamaha mixer this is possible only by using input channels 1 and 2.

**Comment by Stewbow** [comment]
Ahhhhh, I must have the version 1, it doesn't have the red mute button!!!
I'll stick it on Ebay

**Comment by Jazzforever** [comment]
You should wait a little bit, perhaps someone shows up with a better solution than mine, good luck!

**Comment by Jazzforever** [comment]
Another idea: Instead of using mute-buttons, you could try to turn down the channel faders (the pots below the peak-led, not the gain-pots!). Maybe these faders do the same as mute-buttons.

**Comment by QJKX** [comment]
A screengrab of AG06 vs AG06 MK2 in a youtube video.
![image](https://github.com/jamulussoftware/jamulus/assets/23462605/3c5dde11-bed9-42e8-9d98-77985e2f464c)

[AG06 Manual](https://uk.yamaha.com/files/download/other_assets/4/333124/ag06_en_om_a0.pdf)
[AG06 Tech specs](https://usa.yamaha.com/files/download/other_assets/3/333133/ag06_en_ts_b0.pdf)

Tech specs has a block diagram.


The "Monitor Mute" button disconnects Input 1 and Input 2 from Monitor and phones.
All other inputs are always connected.
(You can see Yamaha renamed the button in Mk2 to "Mix Minus Ch1-2".)

So the only way to follow the "golden rule" is to exclusively use Input 1/Input 2 and the "Monitor Mute button".


On the "To PC" switch you can choose "Dry" or "input mix" but not "Loopback".

**Reply by QJKX** [reply]
In case it's not obvious, you can plug a keyboard straight in to Input 1/2. Use the pad buttons.
